include/CFL/CFLAlias.h

```diff
@@ -57,39 +58,41 @@ class CFLAlias : public PointerAnalysis
     /// Start Analysis here (main part of pointer analysis).
     virtual void analyze()
     {
-        PointerAnalysis::initialize();
         GrammarBuilder * gReader = new GrammarBuilder(Options::GrammarFilename);
         CFGNormalizer *normalizer = new CFGNormalizer();
+        CFLGraphBuilder *cflGraphBuilder = new CFLGraphBuilder();
 
         graph = new CFLGraph();
+        /// Assume Read From Const Graph, associate label symbol is hard coded
         if (Options::GraphIsFromDot == false)
         {
-            // Maybe could put in SVFIR Class
-            // In memory Graph does not have string type label
-            Map<std::string, SVF::CFLGraph::Symbol> ConstMap =  {{"Addr",0}, {"Copy", 1},{"Store", 2},{"Load", 3},{"Gep", 4},{"Vgep", 5},{"Addrbar",6}, {"Copybar", 7},{"Storebar", 8},{"Loadbar", 9},{"Gepbar", 10},{"Vgepbar", 11}};
-            ConstraintGraph *consCG = new ConstraintGraph(svfir);
-            svfir->dump("SVFIR");
-            // Can be put in build, copy from general graph
-            graph->label2SymMap = ConstMap;
-            graph->buildBigraph(consCG);
-            graph->dump("PAG");
-            delete consCG;
+            PointerAnalysis::initialize();
+            Map<std::string, SVF::CFLGraph::Symbol> ConstMap =  {{"Addr",0}, {"Copy", 1},{"Store", 2},{"Load", 3},{"Gep_i", 4},{"Vgep", 5},{"Addrbar",6}, {"Copybar", 7},{"Storebar", 8},{"Loadbar", 9},{"Gepbar_i", 10},{"Vgepbar", 11}};
             GrammarBase *generalGrammar = gReader->build(&ConstMap);
+            graph->label2SymMap = ConstMap;
             grammar = normalizer->normalize(generalGrammar);
+            ConstraintGraph *consCG = new ConstraintGraph(svfir);
             graph->setMap(&grammar->terminals, &grammar->nonterminals);
```

Comment:
```
graph->label2SymMap = ConstMap;
graph->setMap(&grammar->terminals, &grammar->nonterminals);
```
These looks very ad-hoc to me. Why do we have this?

Reply:
Initially, since the in memory graph in SVF has kind but not in the form of string,
so I provide the constMap to graph,
And except for those symbol from initial edge label,
during the normalization to binary normal form, it might have more nonterminal generate e.g. X0, X1..
so we also add those symble to graph map.